java/ql/src/Security/CWE/CWE-079/AndroidWebViewSettingsEnabledJavaScript.qhelp

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE qhelp PUBLIC
+ "-//Semmle//qhelp//EN"
+ "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>
+      Enabling JavaScript in an Android WebView allows for the running of JavaScript
+      code in the context of the running application. This opens the possibility for
+      cross-site scripting if the attacker can inject arbitrary JavaScript.
```

Comment:
```suggestion
      Enabling JavaScript in an Android WebView allows the execution of JavaScript
      code in the context of the running application. This creates a vulnerability to
      cross-site scripting.
```

Minor... I wasn't sure if "if the attacker can inject arbitrary JavaScript" was implied or required explicitly (was fairly sure it was implied)?

Reply:
Ok,  I applied this change, but with the modification of "cross-site scripting vulnerability" instead of "vulnerability to cross-site scripting".